modules/integration_azure-sql-elastic-pool/conf/03-dtu-consumption.yaml

```diff
@@ -0,0 +1,21 @@
+---
+module: "Azure SQL Database"
+name: "DTU consumption"
+filtering: "filter('resource_type', 'Microsoft.Sql/servers/databases') and filter('primary_aggregation_type', 'true')"
```

Comment:
Shouldn't it be elasticpools instead of databases ?

Reply:
Good catch !
Fixed.